version.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = "~> 0.18.0"
```

Comment:
This changes the required terraform version, you will want to set this in the provider config.